packages/prices-api/src/savings/api.ts

```diff
@@ -15,30 +15,39 @@ export async function getEvents(page: number, options?: Options) {
   const host = getHost(options)
   const resp = await fetch<Responses.GetEventsResponse>(`${host}/v1/crvusd/savings/events?page=${page}&per_page=10`)
 
-  return {
-    count: resp.count,
-    events: resp.events.map(Parsers.parseEvent),
-  }
+  return { count: resp.count, events: resp.events.map(Parsers.parseEvent) }
 }
 
-export async function getYield(options?: Options) {
+export async function getYield(
+  aggNumber: number = 1,
+  aggUnit: string = 'hour',
+  startDate?: number,
+  endDate?: number,
+  options?: Options,
+) {
   const host = getHost(options)
 
   const { start, end } = getTimeRange({ daysRange: 10 })
 
   const resp = await fetch<Responses.GetYieldResponse>(
-    `${host}/v1/crvusd/savings/yield?agg_number=1&agg_units=hour&start=${start}&end=${end}`,
+    `${host}/v1/crvusd/savings/yield?agg_number=${aggNumber}&agg_units=${aggUnit}&start=${startDate ?? start}&end=${endDate ?? end}`,
```

Comment:
This works, but consider using `URLSearchParams` like in `packages\prices-api\src\lending\api.ts`, it makes building the query string easier if you're dealing with multiple query parameters.